SETUP_SUBMITTER_CMF.md

```diff
@@ -35,12 +35,13 @@ Deadline Cloud Monitor is used to both manage your credentials for submitting jo
 
 ## Deadline Software Installation
 
+- clone or download deadline-cloud-for-unreal-engine either from the release branch or mainline depending on whether you'd like the most recent tested release or all of the most recent commits.
```

Comment:
nit: codeblock for `deadline-cloud-for-unreal-engine`?

Reply:
Hmm I have it in one below for the actual steps, is that what you meant?

Reply:
Sorry, I was referring to having this sentence contain the `deadline-cloud-for-unreal-engine` surrounded by backticks to be consistent with other references to the name of the package.

Reply:
Ahh gotcha, sure!